database/fakeDataGenerator.js

```diff
@@ -0,0 +1,58 @@
+const faker = require('faker');
+const noOfEntries = 100;
+
+let products = [];
```

Comment:
Best practice to not use global variables and instead keep them within a function.

Reply:
k will do.